includes/class-comment-walker.php

```diff
@@ -288,22 +288,14 @@ public static function comment_query( $query ) {
 			return;
 		}
 
-		if ( is_admin() ) {
+		if ( is_admin() || ! is_singular() ) {
 			return;
 		}
 
-		if ( ! is_singular() ) {
+		if ( ! empty( $query->query_vars['type__in'] ) ) {
```

Comment:
Shouldn't we exclude any type specific search?  Not just in? Like type__not_in ? or even just type?

Reply:
good point!